logging/test/logs.test.js

```diff
@@ -16,14 +16,21 @@
 var proxyquire = require('proxyquire').noCallThru();
 var filter = 'severity > ALERT';
 var logName = 'bar';
+var sort = 'field';
+var limit = 1;
+var resource = {
+  type: 'global'
+};
 
 function getSample () {
+  var apiResponseMock = {};
   var entriesMock = [{}];
   var entryMock = {};
   var logMock = {
     entry: sinon.stub().returns(entryMock),
-    write: sinon.stub().callsArgWith(1, null),
-    delete: sinon.stub().callsArgWith(0, null)
+    write: sinon.stub().callsArgWith(1, null, apiResponseMock),
```

Comment:
I assume this is what you wanted to fix in a separate mini-PR. (That's fine by me.)